sbin/newfs_msdos/newfs_msdos.c

```diff
@@ -51,13 +51,32 @@ static u_int argtou(const char *, u_int, u_int, const char *);
 static off_t argtooff(const char *, const char *);
 static void usage(void);
 
+static time_t 
+get_tstamp(const char *b) 
+{
+    struct stat st;
+    char *eb;
+    long long l;
+
+    if (stat(b, &st) != -1) {
```

Comment:
Single line `{ }` is a contentious issue in FreeBSD, and although I like them I'd rather stay with the style that already exists here and matches NetBSD's change.